packages/dds/tree/src/feature-libraries/object-forest/objectForest.ts

```diff
@@ -436,7 +436,9 @@ class Cursor extends SynchronousCursor implements ITreeSubscriptionCursor {
 
     fork(): ITreeSubscriptionCursor {
         assert(this.innerCursor !== undefined, 0x460 /* Cursor must be current to be used */);
-        return new Cursor(this.forest, this.innerCursor.fork());
+        const forked = new Cursor(this.forest, this.innerCursor.fork());
+        this.forest.currentCursors.add(forked);
```

Comment:
I think it would be better to fix this bug by adding to currentCursors in the constructor of this Cursor if its given an inner cursor. That reduces the risk of this bug reoccurring in the future if another call to the constructor is added elsewhere.

Reply:
done